src/launch/launcher.rs

```diff
@@ -71,6 +112,7 @@ impl<'a, U: AsRawFd, V: AsRawFd> Launcher<'a, Started, U, V> {
         LAUNCH_UPDATE_DATA
             .ioctl(self.kvm, &mut cmd)
             .map_err(|e| cmd.encapsulate(e))?;
+        ENC_REG_REGION.ioctl(self.kvm, &KvmEncRegion::new(data))?;
```

Comment:
The other way round please!
```suggestion
        ENC_REG_REGION.ioctl(self.kvm, &KvmEncRegion::new(data))?;

        LAUNCH_UPDATE_DATA
            .ioctl(self.kvm, &mut cmd)
            .map_err(|e| cmd.encapsulate(e))?;
```

Reply:
Same goes for non-SNP, btw.